lib/chartmogul/csv/invoice.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  module CSV
+    class Invoice < ChartMogul::Object
```

Comment:
I don't think this needs to be `ChartMogul::Object`.
```
Invoice = Struct.new(*FIELDS) do
  include/extend ...
```
just works. You can access and write to the fields likewise, so in fact this `writeable_attr` is not necessary. Please simplify.